_schemaV2.graphql

```diff
@@ -14916,6 +14956,12 @@ type Viewer {
   # List of all artwork attribution classes
   artworkAttributionClasses: [AttributionClass]
 
+  # Search for matching artworks by image
+  artworkImageSearch(
```

Comment:
**suggestion**: Uff, naming is hard. I wonder if this is the right name to use here. What do you think of renaming this new root query field to `reverseImageSearch`? I feel like that might capture the intent more clearly.

As is, the name here might imply that this query field is to search for artwork images.

Reply:
> Uff, naming is hard

Oh, I understand you perfectly 😅

![image](https://user-images.githubusercontent.com/3513494/167183644-f8ad29e5-28a6-42d3-a0ab-571fc16f96b0.png)

Reply:
> What do you think of renaming this new root query field to reverseImageSearch?

I like this name 👍

Reply:
Done